aws/assume-role/README.md

```diff
@@ -9,11 +9,39 @@ tasks:
 
   - key: assume-role
     use: aws-cli
```

Comment:
All of the examples still have `aws/assume-role` depending on `aws-cli`. Do they use the CLI anymore? Is this mostly just a convenience so you don't have to specify the CLI dep later when you use the `assume-role` task?